pages/api/mail.ts

```diff
@@ -0,0 +1,52 @@
+// Nodemailer with SES transport, more info here: https://nodemailer.com/transports/ses/
+import { transporter } from "aws/index";
+import { ResponseObject } from "src/models/response";
+
+/**
+ * Sends an email to the targetAddress email from the process.env.EMAIL_FROM email
+ * @param targetAddress the email address we want to send to (string)
+ * @returns nothing
+ */
+export async function sendEmail(targetAddress: string) {
+    await transporter.sendMail({
+        from: process.env.EMAIL_FROM,
+        to: targetAddress, // only for testing, aws ses is in sandbox (recipient email must be verified)
+        subject: "Test Message",
```

Comment:
We should make subject and text to be parameters to this function too but it can be done later in the future

Reply:
I included it anyway :)